core/src/scene/spriteAtlas.cpp

```diff
@@ -24,6 +24,14 @@ bool SpriteAtlas::getSpriteNode(const std::string& _name, SpriteNode& _node) con
     return true;
 }
 
+void SpriteAtlas::updateSpriteNodes(std::shared_ptr<Texture> _texture) {
```

Comment:
Can you add some comment that this is needed because the texture atlas is created before knowing anything about the texture it is associated with (width/height) when fetched over network.